packages/api/src/jobs/certification/survey_record.js

```diff
@@ -0,0 +1,340 @@
+const XlsxPopulate = require('xlsx-populate');
+
+const rp = require('request-promise');
+const surveyStackURL = 'https://app.surveystack.io/api/';
+module.exports = async (submission, exportId) => {
+  const submissionData = await rp({
+    uri: `${surveyStackURL}/submissions/${submission}`,
+    json: true,
+    headers: { Authorization: 'ashaikh@litefarm.org' },
```

Comment:
I feel like we should not be using someone's personal email for this. Also, if we are it should still probably be in a `.env` file as opposed to the code